R/cdfPlot.R

```diff
@@ -171,117 +173,154 @@ cdfPlot <- function(contrastDF,
-            color = group, fill = group,
-            order = order) +
-        # Scale lines tell it to use the actual values, not treat them as factors
-        scale_shape_manual(name = "Group", guide = "none", labels = ssc$group,
-                           values = ssc$symbolShape) +
-        scale_size_manual(name = "Group", guide = "none", labels = ssc$group,
-                          values = ssc$symbolSize) +
-        scale_color_manual(name = "Group", guide = "none", labels = ssc$group,
-                           values = ssc$symbolColor) +
-        scale_fill_manual(name = "Group", guide = "none", labels = ssc$group,
-                          values = ssc$symbolFill) +
-        geom_rect(xmin = 0, xmax = subsetRows,
-                  ymin = 0, ymax = max(contrastDFsubset[[y]]), color = "lightblue",
-                  fill = "lightblue", alpha = 0.2) +
-        geom_point(alpha = alpha)
-
-    # Add Labels
-    if (is.null(xlab)) { # Use colname unless supplied as argument
-        cdfInset <- cdfInset + xlab(xlabel)
-    } else {
-        cdfInset <- cdfInset + xlab(xlab)
-    }
-    if (is.null(ylab)) {
-        cdfInset <- cdfInset + ylab(ylabel)
-    } else {
-        cdfInset <- cdfInset + ylab(ylab)
-    }
-    if (!is.null(insetTitle)) {
+        # Add Labels
         cdfInset <- cdfInset +
+            xlab(xlab) +
+            ylab(ylab) +
             ggtitle(insetTitle)
-    }
 
-    # Adjust font size for inset.
-    factor <- 4/baseFontSize
+        # Adjust font size for inset.
+        factor <- 4/baseFontSize
+        cdfInset <- cdfInset + baseTheme(baseFontSize*factor)
 
-    # Set the font size
-    if (tolower(themeStyle) == "bw") {
-        cdfInset <- cdfInset + theme_bw() + baseTheme(baseFontSize*factor)
-    } else {
-        cdfInset <- cdfInset + theme_grey() + baseTheme(baseFontSize*factor)
-    }
+        # Adjust viewport Y if main Title present
+        vy <- viewportY
+        if (!is.null(title)) {
+            adjust <- (baseFontSize/150)
+            vy <- viewportY - adjust
+        }
 
-    # Adjust viewport Y if main Title present
-    vy <- viewportY
-    if (!is.null(title)) {
-        adjust <- (baseFontSize/150)
-        vy <- viewportY - adjust
-    }
-    # A viewport taking up a fraction of the plot area (upper left)
-    vp <- grid::viewport(width = viewportWidth, height = viewportWidth,
-                         x = viewportX, y = vy,
-                         just = c("left", "top"))
-
-    if (printPlot == TRUE) {
-        print(cdfMain)
-        print(cdfInset, vp = vp)
-    }
+        # A viewport taking up a fraction of the plot area (upper left)
+        vp <- grid::viewport(width = viewportWidth, height = viewportWidth,
+                             x = viewportX, y = vy,
+                             just = c("left", "top"))
 
-    MyList <- list(main = cdfMain, inset = cdfInset, viewport = vp)
-    return(MyList)
+        if (printPlot == TRUE) {
```

Comment:
no need for == TRUE : if (printPlot) ...

Reply:
Updated